style.css

```diff
@@ -75,4 +78,110 @@ a {
     transition: transform 0.2s;
     cursor: pointer;
     margin: 5%;
+  }
+
+.work {
+    display: flex;
+    flex-direction: row;
+    padding: 5%;
+    gap: 5%;
+}
+
+.item{
+    flex: 1;
+}
+
+.text-work1 {
+    color: #344563;
+}
+
+.text-work2 {
+    color: #6b778c;
+}
+
+.little-bottom {
+    border-radius: 6px;
+    background-color: #ebebff;
+    font-size: clamp(8px, 5vw, 12px);
+    padding-left: 1%;
+    padding-right: 1%;
+    color: #6070ff;
+}
+
+.project {
+    align-content: center;
+    padding-left: 1%;
+    padding-right: 1%;
+    border: 3px solid #6070ff;
+    color: #6070ff;
+    display: inline-block;
+    border-radius: 8px;
+    padding-top: 2px;
+}
+
+.menu {
+    height: 20px;
+    width: 20px;
+}
+
+.grid-nav {
+    list-style-type: none;
+    list-style: none;
+    display: grid;
+    align-items: center;
+    justify-content: space-between;
+    grid-template-columns: repeat(4,auto);
+}
+
+.Professional-art {

```

Comment:
This class is not used in the HTML. Consider removing it, following the [YAGNI principle](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)